pkg/controller/common/license/match.go

```diff
@@ -21,14 +22,16 @@ type licenseWithTimeLeft struct {
 // desired license type and the remaining validity period of the license.
 // Returns the license, parent license UID, a bool indicating a match was found and an optional error.
 func BestMatch(
+	minVersion *version.Version,
```

Comment:
Should we update the godoc to mention why there is an additional parameter ? (not 💯 % it's really worth it but it can help when we read the code)